gate-teamware/values.yaml

```diff
@@ -377,13 +392,31 @@ backup:
   # Backup schedule.  Keys in the map are the name of the backup schedule
   # (e.g. daily, weekly), values are the cron expression defining the schedule.
   # The actual backup files will be placed at
-  # s3://<bucketName>/<keyPrefix>/<schedule-name>/<database-name>_<timestamp>.sql.gz
+  # store:<uploadPrefix>/<schedule-name>/<database-name>_<timestamp>.sql.gz
   #
   # Note that the cron job creates new backups but does not expire old ones -
   # if you want old backups to expire (or be moved to cheaper storage like
-  # infrequent access or Glacier) then you should configure that at the bucket
-  # level using S3 lifecycle rules.
+  # infrequent access or Glacier) then you should configure that using the
+  # storage provider's own mechanisms, e.g. S3 lifecycle rules.
   schedule: {}
   # daily:   "13 01 * * *"
   # weekly:  "13 02 * * 0"
   # monthly: "13 00 1 * *"
+
+  # extra environment variable settings to be passed to the backup container
+  extraEnv: []
+  # extra command line arguments to be passed to the backup container (and
+  # from there through to the pg_dump command)
+  extraArgs: []
+  # extra volume definitions added to the backup pod, e.g. an rclone
+  # configuration file, private key for authentication to an sftp datastore,
+  # TLS CA certificates, etc. etc.
+  extraVolumes: []
+  # volumeMounts entries to mount the extra volumes into the backup container
+  extraVolumeMounts: []
```

Comment:
If you're _not_ using S3 then it's up to you to pass the appropriate `RCLONE_CONFIG_STORE_*` environment variables and any other required configuration (e.g. mounting an SSH private key from a secret, for authentication to an SFTP server).